docs/markdown/foundations/content_standards/voice.md

```diff
@@ -3,13 +3,10 @@ title: Pinterest Voice
 description: The Pinterest voice is our brand personality—it’s a part of everything we do. When writing at Pinterest, keep these guiding words in mind
 fullwidth: true
 ---
-
-**Simplify ideas to make the complex clear**
+## Simplify ideas to make the complex clear
```

Comment:
nit: I feel like these have a lot of text for a h2. I think we made an explicit change to use bold when they were written

Would it make more sense to create a section called Principles and then make these H3s under it?

Reply:
good feedback! I would prefer to do this as a v2 since stakeholders have already approved. There will be additions in October, so we can address then.

Reply:
Oh, wait. These are for content standards. those were created by the content standards team. we can also take these up later since there are some updates to make to this section too.